examples/tutorials/pong/steps/step6/main.py

```diff
@@ -0,0 +1,109 @@
+import kivy
+kivy.require('1.1.1')
+
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.properties import NumericProperty, ReferenceListProperty,\
+    ObjectProperty
+from kivy.vector import Vector
+from kivy.clock import Clock
+
+
+class BounceManager():
```

Comment:
inherit from `object` until we drop the support for Py2, otherwise:

```python
class A(): pass

class B(object): pass

type(A)  # classobj
type(B)  # type
```

Reply:
Ok